guide/secure_restful_service/secure_order_mgt_service.bal

```diff
@@ -19,20 +37,25 @@ import ballerina/http;
 //    name:"ballerina-guides-secure-restful-service"
 //}
 //
-//@kubernetes:Deployment {
-//    image:"ballerina.guides.io/secure_restful_service:v1.0",
-//    name:"ballerina-guides-secure-restful-service"
+//@docker:Config {
+//    registry:"ballerina.guides.io",
+//    name:"secure_restful_service",
+//    tag:"v1.0"
 //}
-
-@final string regexInt = "\\d+";
-@final string regexJson = "[a-zA-Z0-9.,{}:\" ]*";
-
-http:AuthProvider basicAuthProvider = {
-    scheme: "basic",
-    authStoreProvider: "config"
-};
-endpoint http:SecureListener listener {
+//
```

Comment:
```suggestion

```

Reply:
This is the only issue I saw. Travis is failing due to older Ballerina version used. That will be fixed separately.

Reply:
Shall we remove `install-ballerina.sh` with this PR.